README.md

```diff
@@ -1,3 +1,373 @@
+
+- `key`: path to where the metadata should be stored in the run.
+- `value`: the piece of metadata to log.
+
+For example, `{"parameters/learning_rate": 0.001}`. In the field path, each forward slash `/` nests the field under a namespace. Use namespaces to structure the metadata into meaningful categories.
+
+__Parameters__
+
+| Name          | Type                                               | Default | Description                                                               |
+|---------------|----------------------------------------------------|---------|---------------------------------------------------------------------------|
+| `timestamp`   | `datetime`, optional                               | `None`  | Time of logging the metadata. |
+| `data`      | `Dict[str, Union[float, bool, int, str, datetime, list, set]]`, optional  | `None` | Dictionary of configs or other values to log. Available types: float, integer, Boolean, string, and datetime. |
+
+__Examples__
+
+Create a run and log metadata:
+
+```python
+from neptune_scale import Run
+
+with Run(...) as run:
+    run.log_configs(
+        data={
+            "parameters/learning_rate": 0.001,
+            "parameters/batch_size": 64,
+        },
+    )
+```
+
+### `log_metrics()`
+
+Logs the specified metrics to a Neptune run.
+
+You can log metrics representing a series of float values. Pass the metadata as a dictionary `{key: value}` with
+
+- `key`: path to where the metadata should be stored in the run.
+- `value`: the piece of metadata to log.
+
+For example, `{"metrics/accuracy": 0.89}`. In the field path, each forward slash `/` nests the field under a namespace. Use namespaces to structure the metadata into meaningful categories.
+
+__Parameters__
+
+| Name          | Type                                               | Default | Description                                                               |
+|---------------|----------------------------------------------------|---------|---------------------------------------------------------------------------|
+| `step`        | `Union[float, int]`, optional                      | `None`  | Index of the log entry. Must be increasing. If not specified, the `log_metrics()` call increments the step starting from the highest already logged value. **Tip:** Using float rather than int values can be useful, for example, when logging substeps in a batch. |
+| `timestamp`   | `datetime`, optional                               | `None`  | Time of logging the metadata. |
+| `data`     | `Dict[str, float]`, optional                       | `None`  | Dictionary of metrics to log. Each metric value is associated with a step. To log multiple metrics at once, pass multiple key-value pairs. |
+
+__Examples__
+
+Create a run and log metrics:
+
+```python
+from neptune_scale import Run
+
+with Run(...) as run:
+    run.log_metrics(
+        data={"loss": 0.14, "acc": 0.78},
```

Comment:
Let's provide the `step` value as well as it almost doesn't make sense to use this method without it.